src/core/rendering.js

```diff
@@ -796,23 +796,23 @@ p5.prototype.clearDepth = function(depth) {
  * function setup() {
  *   createCanvas(100, 100);
  *
- *   background(200);
+ *   background(180);
  *
  *   // Set the blend mode.
- *   blendMode(ADD);
+ *   blendMode(HARD_LIGHT);
```

Comment:
I noticed that you removed two example sketches:
1. blendMode(ADD)
2. blendMode(DARKEST)

While the colors didn’t show a noticeable difference, I think we could keep these examples by using a different set of colors. What do you think?

Reply:
Hey @perminder-17 !! I have added these two as well, is there any sort of order we need to follow while adding all the blendmodes?

Reply:
> Hey @perminder-17 !! I have added these two as well, is there any sort of order we need to follow while adding all the blendmodes?

No, i don't think so. Its just that we need to help users learn about blendMode() so it could be in any order.

Reply:
sure